hc/front/tests/test_my_checks.py

```diff
@@ -58,3 +58,19 @@ def test_it_shows_amber_check(self):
 
         # Mobile
         self.assertContains(r, "label-warning")
+
+    def test_failed_checks_view(self):
```

Comment:
This is good but since this is a failed checks view, shouldnt you assert that failed checks actually exist, ie `contains "icon-down"`?